setup.py

```diff
@@ -13,9 +17,30 @@
         'ruamel.yaml>=0.15',
     ],
     python_requires='>=3.4',
-    author='Repo2docker contributors',
-    author_email='yuvipanda@gmail.com',
+    author='Project Jupyter Contributors',
+    author_email='jupyter@googlegroups.com',
+    url='https://repo2docker.readthedocs.io/en/latest/',
+    project_urls = {
+        'Documentation': 'https://repo2docker.readthedocs.io',
+        'Funding': 'https://jupyter.org/about',
+        'Source': 'https://github.com/jupyter/repo2docker/',
+        'Tracker': 'https://github.com/jupyter/repo2docker/issues',
+    },
+    # this should be a whitespace separated string of keywords, not a list
+    keywords="reproducible science environments docker",
+    description = "Repo2docker: Turn git repositories into Jupyter enabled Docker Images",
```

Comment:
Git isn't required, which makes the repo in repo2docker a misnomer. However, I can't think of any alternative right now, so I'm ok with this :)

Reply:
Maybe: "Turn a set of files, like a git code repository, into Jupyter enabled Docker Images.